plugins/inputs/mqtt_consumer/mqtt_consumer.go

```diff
@@ -16,11 +16,12 @@ import (
 )
 
 type MQTTConsumer struct {
-	Servers  []string
-	Topics   []string
-	Username string
-	Password string
-	QoS      int `toml:"qos"`
+	Servers           []string
+	Topics            []string
+	Username          string
+	Password          string
+	QoS               int `toml:"qos"`
+	ConnectionTimeout int `toml:"connection_timeout"`
```

Comment:
Lets use `internal.Duration` for the timeout.  [Here is an example](https://github.com/influxdata/telegraf/blob/master/plugins/outputs/influxdb/influxdb.go#L34).

Reply:
Good idea, Thanks!